frontend/packages/metal3-plugin/src/components/modals/RestartHostModal.tsx

```diff
@@ -41,19 +39,12 @@ const RestartHostModal = ({
     [host, close, handlePromise],
   );
 
-  const text = nodeName
-    ? t(
-        'metal3-plugin~The bare metal host {{name}} will be restarted gracefully after all managed workloads are moved.',
```

Comment:
WDYT of adding back a tiny bit of `<p>` text above the warning that reads:

> The host will be powered off and on again.

Reply:
I know it's a bit obvious 🙂 but I think we typically have _something_ in the body of modals explaining what's going to happen before warnings/alerts.

Reply:
sure, added!